### DISCUSSION TITLE
Tycho 2.5.0 release

### DISCUSSION THREAD

**Discussion by akurtakov** [eclipse-tycho/tycho#295] [Announcements]
We plan to release Tycho 2.5.0 next week. For details of new features and bugfixes, see [release notes](https://github.com/eclipse/tycho/blob/tycho-2.5.x/RELEASE_NOTES.md).
Please help by testing the SNAPSHOTS build. To use it, change your tycho version to 2.5.0-SNAPSHOT and add the following snippet to your pom.

```
<pluginRepositories>
    <pluginRepository>
      <id>tycho-snapshots</id>
      <url>https://repo.eclipse.org/content/repositories/tycho-snapshots/</url>
    </pluginRepository>
</pluginRepositories>
```

We plan to promote this release in one week unless major regressions are found.

Regards,
Tycho team

**Comment by laeubi** [comment]
Checked successful with several projects of my company.

**Comment by laeubi** [comment]
m2e works now also with tycho 2.5.x https://github.com/eclipse-m2e/m2e-core/pull/334#issuecomment-926462675

**Comment by cdietrich** [comment]
no problems with Xtext

**Comment by mickaelistria** [comment]
Hi all,

A lot of important change have happened in Tycho 2.5 since the announcement. Please make sure you try a recent build to verify no regression was introduced in the meantime.
If no blocker is reported, we'll release Tycho 2.5 on Wednesday.

**Comment by mickaelistria** [comment]
Tycho 2.5.0 has been released and is available from Maven Central repository.

🆕 https://github.com/eclipse/tycho/blob/master/RELEASE_NOTES.md#250
🏷️ https://github.com/eclipse/tycho/tree/tycho-2.5.0
👔 https://projects.eclipse.org/projects/technology.tycho/releases/2.5.0
🙏 contributors who contributed patches for this release:
Alexander Kurtakov
András Péteri
Christoph Läubrich
Guillaume Nodet
Hannes Wellmann
Jérémie Bresson
Joe Shannon
Johan Compagner
Lorenzo Bettini
Michael Keppler
Mickael Istria
Olivier Lamy
and thanks and to everyone who helped us with testing the staged version.

Regards,
